src/wallet.rs

```diff
@@ -206,21 +202,23 @@ pub async fn get_balance(
 mod tests {
     use super::*;
     use ctor::ctor;
-    use dotenv::dotenv;
+    use dotenv;
 
     #[cfg(test)]
     #[ctor]
     fn init() {
-        dotenv().ok();
+        dotenv::from_filename(".env.example").ok();
     }
 
     #[tokio::test]
     async fn test_web3_get_balance() {
-        let balance = web3_get_balance("ethereum", "0xEFb616A5cdE977f87A9878EbEC0b23c655bac762")
-            .await
-            .unwrap();
-        println!("ETH: {}", balance);
-        assert_eq!(Decimal::new(0, 0), balance);
+        for chain in ["ethereum", "polygon", "avalanche", "bsc"].iter() {
+            let balance = web3_get_balance(chain, "0xEFb616A5cdE977f87A9878EbEC0b23c655bac762")
+                .await
+                .unwrap();
+            println!("ETH: {}", balance);
```

Comment:
remove `ETH:` ?

Reply:
in case developers don't open verbose option, should be fine.  
any word is acceptable.

Reply:
or rename it to balance: {} ?